site/app/templates/officeHoursQueue/CurrentQueue.twig

```diff
@@ -62,6 +63,13 @@
           {% else %}
             <td>{{ viewer.timeToHM(entry['time_in']) }}</td>
           {% endif %}
+
+          {% if entry['time_paused_start'] == NULL or entry['current_state'] == 'being_helped' %}
+            <td>---</td>
+          {% else %}
+            <td class="help_timer" help_time="{{ viewer.timeToISO(entry['time_paused_start']) }}"></td>
```

Comment:
In testing this through multiple pauses and unpauses, this column (and also the similar value on the user's UI) was always displaying "∞": https://www.loom.com/share/0c7d7c658c6e487bbc96ffb47f30c965

Reply:
Interesting. My first guess would be some time zone issue that I may have missed, but I already did some testing for this so this may not be the case.

Given that I haven't been able to recreate this, would you mine commenting out this piece of code right here? This should give us some insight on what is going on.
https://github.com/Submitty/Submitty/blob/37914d84cb47cd3ca483661cf29a89820fb87172/site/app/templates/officeHoursQueue/QueueFooter.twig#L128-L130

Also, could you try helping the student and see if that timer is working properly?

Reply:
On getting back into EST, it now works as expected (it showed inf when I was in UTC last week). My guess is that there is indeed some timezone bug, but could not easily reproduce at the moment. Resolving this for now.